utils/utils.go

```diff
@@ -49,3 +53,11 @@ func InitReporterHttpClient() httputils.IHttpClient {
 	}
 	return &http.Client{}
 }
+
+func ExtractImageID(imageID string) string {
+	return strings.TrimPrefix(imageID, dockerPullableURN)
+}
+
+func GenerateInstanceID(parentApiVersion, namespace, kind, name, resourceVersion, containerName string) string {
```

Comment:
add test for this function so that if someone changes the logic it will break the test